lib/open_api_spex/plug/swagger_ui.ex

```diff
@@ -112,7 +112,11 @@ defmodule OpenApiSpex.Plug.SwaggerUI do
         plugins: [
           SwaggerUIBundle.plugins.DownloadUrl
         ],
-        layout: "StandaloneLayout"
+        layout: "StandaloneLayout",
+        requestInterceptor: function(request){
+          request.headers["x-csrf-token"] = "<%= Plug.CSRFProtection.get_csrf_token() %>";
```

Comment:
How does this behave when the CSRFProtection plug isn't being used in the plug pipeline before the SwaggerUI plug?